cli/src/commands/router/commands/fetch.ts

```diff
@@ -70,31 +75,34 @@ export default (opts: BaseCommandOptions) => {
       headers,
       body: requestBody,
     });
-    if (response.status !== 200) {
-      const latestConfigResp = await opts.client.platform.getLatestValidRouterConfig(
-        {
-          graphName: name,
-          namespace: options.namespace,
-        },
-        {
-          headers: baseHeaders,
-        },
-      );
-
-      if (latestConfigResp.response?.code !== EnumStatusCode.OK) {
-        console.log(`${pc.red(`No router config could be fetched for federated graph ${pc.bold(name)}`)}`);
-        if (resp.response?.details) {
-          console.log(pc.red(pc.bold(resp.response?.details)));
-        }
+
+    const body = await response.text();
+    const signature = response.headers.get('X-Signature-SHA256');
```

Comment:
Should the header name be a const and re-used across all methods?

Reply:
It is used only here in the cli.